### DISCUSSION TITLE
Where to find the official Veramo Discord

### DISCUSSION THREAD

**Discussion by JonasHiltl** [decentralized-identity/veramo#684] [General]
I have already tried for some time to find the Veramo Discord but couldn't find a Link. Does Veramo have an official Discord to engage with fellow people interested/using Veramo?
I would also suggest to improve the discoverability of the Discord on your website, by maybe adding it in the footer of [veramo.io](https://veramo.io/)

**Comment by mirceanis** [comment]
Hi @JonasHiltl, we have just started a discord server and you are free to join here: https://discord.gg/huwyNfVkhe.
